### DISCUSSION TITLE
Timezone offset on Next.js server is incorrect

### DISCUSSION THREAD

**Discussion by mattpodolak** [vercel/next.js#24815] [Help] [answered]
### What version of Next.js are you using?

10.1.3

### What version of Node.js are you using?

14.5.0

### What browser are you using?

Chrome

### What operating system are you using?

Windows

### How are you deploying your application?

npm run dev

### Describe the Bug

Trying to parse a UTC date and the timezone offset of the server is incorrect.

```
  lastUpdated // UTC date
  console.log(lastUpdated); // 2021-05-02 10:00:06
  console.log(Date.parse(lastUpdated)); // 1619964006000
  const date = new Date(Date.parse(lastUpdated));
  console.log(date); //2021-05-02T14:00:06.000Z
  console.log(date.getTimezoneOffset()); //240
```

Expected timezone offset is -240, so now all my dates are wrong by 8 hours. Tried passing a TZ environment variable with  'America/New_York' but it did not change anything

### Expected Behavior

```
  lastUpdated // UTC date
  console.log(lastUpdated); // 2021-05-02 10:00:06
  console.log(Date.parse(lastUpdated)); // 1619964006000
  const date = new Date(Date.parse(lastUpdated));
  console.log(date); //2021-05-02T06:00:06.000Z
  console.log(date.getTimezoneOffset()); //-240
```

### To Reproduce

Run a next.js server

**Comment by wwwdepot** [comment, marked as answer]
> The number of minutes returned by getTimezoneOffset() is positive if the local time zone is behind UTC, and negative if the local time zone is ahead of UTC. For example, for UTC+10, -600 will be returned.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getTimezoneOffset

**Reply by mattpodolak** [reply]
Thanks for the reply, it was confusing as it was also adding 240 minutes to the date instead of subtracting. I'm guessing this is because Date.parse assumes the date string is in the local timezone and converts it to UTC.
